packages/lesmis/src/utils/useDashboardDropdownOptions.js

```diff
@@ -62,6 +53,8 @@ export const useDashboardDropdownOptions = () => {
 
   return {
     dropdownOptions,
+    favouriteDropdownOption: dropdownOptions.find(({ value }) => value === 'favourites'),
```

Comment:
One more spot we're using the `'favourites'` magic string

Reply:
Good spotting!